client/src/pages/TempHome.js

```diff
@@ -0,0 +1,16 @@
+import { BrowserRouter , Switch, Route } from "react-router-dom";
+import Profile from './Profile';
+
+
+
+const Home = () => {
+    return (
+        <BrowserRouter>
+          <Switch>
+            <Route path="/profile" component={Profile}/>
+          </Switch>
+        </BrowserRouter>
+    );
+}
```

Comment:
why don't we have this directly in `App.js`?

Reply:
there is no special reason just because I cloned the Carlos's branch to start the code (Bonnie told me it is better to clone this branch). It was structured like this. I thought there was a reason behind it so I keep it. I will change it. Thanks.